modules/ROOT/pages/jdbc-tracing.adoc

```diff
@@ -195,10 +195,11 @@ The Oracle JDBC and Oracle JDBC debug drivers support the `java.util.logging` li
 </jdbcDriver>
 ----
 
-Add the following option to the `jvm.options` file to ensure that all trace is produced.
+For Oracle 23c and later, add the following option to the `jvm.options` file to enable trace.
+To avoid the performance impact of logging trace strings the Oracle JDBC driver will not log trace without this property set to true.
```

Comment:
```suggestion
To avoid the performance impact of logging trace strings, the Oracle JDBC driver does not log trace unless this property is set to true.
```